src/Notifications.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+
+interface NotificationsProps {
+  removeTodoIsClicked: boolean,
+  setRemoveTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  addTodoIsClicked: boolean,
+  editTodoIsClicked: boolean,
+  setEditTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+  setAddTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const Notifications: React.FC<NotificationsProps> = ({
+  removeTodoIsClicked,
+  setRemoveTodoIsClicked,
+  addTodoIsClicked,
+  editTodoIsClicked,
+  setEditTodoIsClicked,
+  setAddTodoIsClicked,
+}) => {
+  const [hideNotificationIsClicked,
+    setHideNotificationIsClicked] = useState(false);
+
+  const hideNotification = () => {
+    setHideNotificationIsClicked(true);
+    setRemoveTodoIsClicked(false);
+    setAddTodoIsClicked(false);
+    setEditTodoIsClicked(false);
+  };
+
+  let notificationText = '';
+
+  if (removeTodoIsClicked) {
+    notificationText = 'Unable to delete a todo';
+  } else if (addTodoIsClicked) {
+    notificationText = 'Unable to add a todo';
+  } else if (editTodoIsClicked) {
+    notificationText = 'Unable to edit a todo';
+  }
+
+  useEffect(() => {
+    if (!hideNotificationIsClicked) {
+      const timer = setTimeout(() => {
+        hideNotification();
+      }, 3000);
+
+      return () => clearTimeout(timer);
+    }
+
+    return undefined;
```

Comment:
You can remove the 'return undefined;' line as it's not necessary.

Reply:
WARNING in [eslint] 
src/Notifications.tsx
  Line 40:16:  Expected to return a value at the end of arrow function  consistent-return